### PR TITLE
Added root path options

### PR DESCRIPTION
**Pull request by Bladrak**
This PR adds the following options:
* ``S3_LOADER_ROOT_PATH``
* ``STORAGE_AWS_STORAGE_ROOT_PATH``

Refactoring will be done in #16

See also #11

### CODE REVIEW

**Comment by dhardy92** [comment]
Seams good to me reading it (no time to test  it for now).
Do you think you can add vows tests for this ?

**Comment by Bladrak** [comment]
I'll try to do that yes :)

**Comment by Bladrak** [comment]
@dhardy92 I've added vows for the other storage, quite a lot has changed :)

**Comment by dhardy92** [comment]
I am going off for 3 weeks no time to test it myself but seams good.
no tests for new loader setting ?

**Comment by Bladrak** [comment]
Loader is a lot more painful to test with pyVows as it's a module and doesn't return anything. We would have to fully load up thumbor to test it.

**Comment by Bladrak** [comment]
See #18 for this coverage :)

**Comment by Bladrak** [comment]
@masom if you have some time to check this PR :) Coverage is now complete, but I'd like the validation of someone nonetheless (given @dhardy92 is on holidays for now) :)

File: Readme.md [hunk 1]
```diff
@@ -38,9 +42,20 @@ Additional Configuration values used:
 
     # alternatively: set a fixed bucket, no need for bucket name in Image-Path
     S3_LOADER_BUCKET = 'thumbor-images'
+    # A root path for loading images, useful if you share the bucket
+    S3_LOADER_ROOT_PATH = 'source-images'
 
     # configuration settings specific for the storages
 
+    STORAGE_BUCKET = 'thumbor-images'
```
Comment (masom) [inline Readme.md]: It would be nice at some point to move all config values to `AWS_` or `S3_` prefix to prevent collisions.

Comment (Bladrak) [inline Readme.md]: Created issue #20 for that :)

**Pull request merged**